public/content/documentation/web/page-level/form.md

```diff
@@ -1,80 +1,598 @@
+- Autofill attributes to help customers complete fields with less effort.
+- Using inputmode="numeric" brings up the numeric keyboard on mobile devices making entry easier.
+
+<example>
+    <form aria-label="Payment information">
+        <fieldset>
+            <legend>Enter your payment information</legend>
+            <div class="card-number-container">
+                <label for="card-number">Card Number</label>
+                <button type="button" class="cc-camera secondary">
+                   <span class="hidden">Use camera to</span> 
+                   Scan
+                   <span class="hidden">card number</span>
+                </button>
+                <input type="text" 
+                       name="cardnumber" 
+                       id="card-number" 
+                       autocomplete="cc-number"
+                       inputmode="numeric"
+                       pattern="[0-9]*"
+                       aria-describedby="description-card-number"
+                       required>
+                <div id="description-card-number" class="hint secure-icon">
+                  Secure form protected by 8 bit encryption
+                </div>
+            </div>
+        </fieldset>
+        <fieldset>
+            <legend>Expiration <span aria-hidden="true">(MM YY)</span></legend>
+            <div class="expiry-container">
+                <label class="hidden" for="cc-exp-month">Expiration Month (MM)</label>
+                <input type="text"
+                       maxlength="2"  
+                       name="ccmonth" 
+                       id="cc-exp-month"
+                       inputmode="numeric"
+                       autocomplete="cc-exp-month"
+                       pattern="[0-9]*"
+                       required>
+                <label class="hidden" for="cc-exp-year">Expiration Year (YY)</label>
+                <input type="text"
+                      maxlength="2"  
+                      name="ccyear" 
+                      id="cc-exp-year"
+                      inputmode="numeric"
+                      autocomplete="cc-exp-year"
+                      pattern="[0-9]*"
+                      required>
+            </div>
+        </fieldset>
+        <label for="cc-name">Name on card</label>
+        <input type="text"  
+               name="ccname" 
+               id="cc-name" 
+               autocomplete="cc-name" 
+               maxlength="19" 
+               required>
+        <label for="cc-cvc">Security code</label>
+        <input type="text" 
+               maxlength="5" 
+               name="cvc" 
+               id="cc-cvc"  
+               autocomplete="cc-csc"
+               inputmode="numeric"
+               pattern="[0-9]*"
+               aria-describedby="description-cc-cvc"
+               required>
+        <div id="description-cc-cvc" class="hint">
+            <button type="button" class="inline-link">What's a security code?</button>
```

Comment:
add class="Magentaa11y-button Magentaa11y-button--primary" to make the button pink